src/suiryoku/locomotion/process/locomotion.cpp

```diff
@@ -819,11 +819,26 @@ bool Locomotion::position_kick_range_pan_tilt(const keisan::Angle<double> & dire
     return true;
   }
 
-  auto target_tilt = 0.5 * (position_min_range_tilt + position_max_range_tilt);
+    // y movement
+  left_kick = precise_kick ? left_kick : pan > 0.0_deg;
```

Comment:
No need to set `left_kick` if `precise_kick` is true:
```suggestion
  if (!precise_kick) left_kick = pan > 0.0_deg;
```
It's up to you though for deciding which style you like.